[2025-06-27T09:00:10.236+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_relatorio_revendas_diario.separed_data scheduled__2025-06-26T09:00:00+00:00 [queued]>
[2025-06-27T09:00:10.253+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_relatorio_revendas_diario.separed_data scheduled__2025-06-26T09:00:00+00:00 [queued]>
[2025-06-27T09:00:10.254+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-06-27T09:00:10.324+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): separed_data> on 2025-06-26 09:00:00+00:00
[2025-06-27T09:00:10.336+0000] {standard_task_runner.py:60} INFO - Started process 342200 to run task
[2025-06-27T09:00:10.346+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'DAG_relatorio_revendas_diario', 'separed_data', 'scheduled__2025-06-26T09:00:00+00:00', '--job-id', '267', '--raw', '--subdir', 'DAGS_FOLDER/DagEnviarRelatorioDesempenho.py', '--cfg-path', '/tmp/tmp7q1_k8bl']
[2025-06-27T09:00:10.352+0000] {standard_task_runner.py:88} INFO - Job 267: Subtask separed_data
[2025-06-27T09:00:10.484+0000] {task_command.py:423} INFO - Running <TaskInstance: DAG_relatorio_revendas_diario.separed_data scheduled__2025-06-26T09:00:00+00:00 [running]> on host 99f5714845f7
[2025-06-27T09:00:10.754+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='DAG_relatorio_revendas_diario' AIRFLOW_CTX_TASK_ID='separed_data' AIRFLOW_CTX_EXECUTION_DATE='2025-06-26T09:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-26T09:00:00+00:00'
[2025-06-27T09:00:10.990+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-27T09:00:11.013+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=DAG_relatorio_revendas_diario, task_id=separed_data, execution_date=20250626T090000, start_date=20250627T090010, end_date=20250627T090011
[2025-06-27T09:00:11.164+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-27T09:00:11.223+0000] {taskinstance.py:3280} INFO - 3 downstream tasks scheduled from follow-on schedule check
